modules/vba_documents/app/models/vba_documents/upload_submission.rb

```diff
@@ -6,6 +6,9 @@ class UploadSubmission < ActiveRecord::Base
     include SentryLogging
 
     IN_FLIGHT_STATUSES = %w[received processing].freeze
+    STATSD_PREFIX = 'VBADocument_UploadSubmission'
```

Comment:
code base seems to favor STATSD_KEY_PREFIX throughout -- would prefer we match unless there's a reason not to.. also see if there are any examples in the vba related code.. the prefixes I see tend to take a namespace approach like api.vba.document_upload .. but see what you find in the code base